octoprint_mrbeam/printing/comm_acc2.py

```diff
@@ -2079,29 +2079,50 @@ def _replace_intensity(self, cmd):
         obj = self._regex_intensity.search(cmd)
         if obj is not None:
             intensity_limit = int(self._laserCutterProfile["laser"]["intensity_limit"])
+            max_correction_factor = float(
+                self._laserCutterProfile["laser"]["max_correction_factor"]
+            )
             intensity_cmd = cmd[obj.start() : obj.end()]
             parsed_intensity = int(intensity_cmd[1:])
 
             # Limit GCode input (in case users enter a too high value in the gcode)
             self._current_intensity = parsed_intensity
             if self._current_intensity > intensity_limit:
+                self._logger.warn(
+                    "gcode intensity higher as allowed max, will limit to max value - %s => %s",
+                    self._current_intensity,
+                    intensity_limit,
+                )
```

Comment:
Same here, please do not log stuff for every single gcode command processed. This part of the code needs to be more lean. Refrain from doing IO with the SD card if you could.